src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int FIGURE_ARRAY_SIZE = 6;
```

Comment:
Avoid using static methods or fields unless they are constants or utilities. In this case, FIGURE_ARRAY_SIZE can be a non-static field if it's intended to be configurable for instances of Main.